modules/client-configuration/pages/tshoot-clients.adoc

```diff
@@ -168,8 +168,23 @@ Do not interrupt the process with kbd:[Ctrl+C].
 
 == Mounting /tmp with noexec
 
-Salt runs remote commands from [filename]``/tmp`` on the client's file system.
-Therefore you must not mount [filename]``/tmp`` with the [option]``noexec`` option.
+Salt runs remote commands from [path]``/tmp`` on the client's file system.
+Therefore you must not mount [path]``/tmp`` with the [option]``noexec`` option.
+The other way to solve this issue is to override temporary directory path with [literal]``TMPDIR`` environment variable specified for the Salt service to make it pointing to the directory with no [option]``noexec`` option set.
+It is recommended to use systemd drop-in configuration file [path]``/etc/systemd/system/venv-salt-minion.service.d/10-TMPDIR.conf`` if Salt Bundle is used, or [path]``/etc/systemd/system/salt-minion.service.d/10-TMPDIR.conf`` if classic [package]``salt-minion`` is used on the client.
+The example of the drop-in configuration file content:
+----
+[Service]
+Environment=TMPDIR=/var/tmp
+----
+
+
+
+== Mounting /var/tmp with noexec
+
+Salt SSH is using [path]``/var/tmp`` to deploy Salt Bundle to and execute the Salt on the client with the bundled Python.
+Therefore you must not mount [path]``/var/tmp`` with the [option]``noexec`` option.
+It is not possible to bootstrap the clients, which have [path]``/var/tmp`` mounted with [option]``noexec`` option, via {webui} as the bootstrap process is using Salt SSH to reach a client.
```

Comment:
```suggestion
It is not possible to bootstrap the clients, which have [path]``/var/tmp`` mounted with [option]``noexec`` option, with the {webui} because the bootstrap process is using Salt SSH to reach a client.
```